README.md

```diff
@@ -29,14 +29,18 @@ KALEIDOSCOPE_INIT_PLUGINS(LEDRainbowEffect, LEDRainbowWaveEffect);
 
 void setup() {
   Kaleidoscope.setup();
+
+  LEDRainbowEffect.brightness(150);
+  LEDRainbowWaveEffect.brightness(150);
+  LEDRainbowWaveEffect.delay(50);
 }
 ```
 
 ## Plugin methods
 
 The plugin provides two objects: `LEDRainbowEffect`, and `LEDRainbowWaveEffect`,
-neither of which have any public methods or properties, outside of those
-provided by all LED modes.
+both of which provide methods to set the delay time (animation speed)
+and effect brightness, called `delay` and `brightness` respectively.
```

Comment:
Would it be possible for you to adjust this section to be more like in [other plugins](https://github.com/keyboardio/Kaleidoscope-OneShot#plugin-methods)?